tests/python/unittest/test_numpy_op.py

```diff
@@ -1162,6 +1162,63 @@ def test_np_broadcast_arrays():
     pass
 
 
+@with_seed()
+@npx.use_np
+def test_np_vstack():
+    class TestVstack(HybridBlock):
+        def __init__(self):
+            super(TestVstack, self).__init__()
+        
+        def hybrid_forward(self, F, a, *args):
+            return F.np.vstack([a] + list(args))
+    
+    def g(data):
+        return _np.ones_like(data)
+
+    configs = [
+        ((), (), ()),
+        ((2), (2), (2)),
+        ((0), (0), (0)),
+        ((2, 2), (3, 2), (0, 2)),
+        ((2, 3), (1, 3), (4, 3)),
+        ((2, 2, 2), (3, 2, 2), (1, 2, 2)),
+        ((0, 1, 1), (4, 1, 1), (5, 1, 1)),
+        ((2), (0, 2), (2, 2))
+    ]
+    types = ['float16', 'float32', 'float64', 'int8', 'int32', 'int64']
+    for config in configs:
+        for hybridize in [True, False]:
+            for dtype in types:
+                print('config = {}'.format(config))
```

Comment:
get rid of the print if it was for debugging purpose.

Reply:
removed. @haojin2